sleap/gui/commands.py

```diff
@@ -2029,6 +2033,34 @@ def ask(context: CommandContext, params: dict) -> bool:
 
         return len(filename) > 0
 
+class LinkVideoToSession(EditCommand):
+    topics = [UpdateTopic.video]
+
+    @staticmethod
+    def do_action(context: CommandContext, params: dict):
+        video = context.state["video"]
+        session = context.state["session"]
+
+        if video is None:
+            raise ValueError("No video selected.")
+
+        if session is None:
+            raise ValueError("No session selected.")
+
+        video.recording_session = session
+
+    @staticmethod
+    def ask(context: CommandContext, params: dict) -> bool:
+        """Shows gui for adding video to project."""
+        if context.state["video"] is None:
+            QtWidgets.QMessageBox(text="No video selected.").exec_()
+            return False
+
+        if context.state["session"] is None:
+            QtWidgets.QMessageBox(text="No session selected.").exec_()
+            return False
+
+        return True
```

Comment:
Remove - the `ask` method is intended to be (although has not always been) used to ask the user for additional input through the GUI (i.e. the name of a file, "are you sure", etc.).
```suggestion
```